Makefile

```diff
@@ -15,13 +17,16 @@ ifeq ($(OS),Darwin)
 endif
 
 ifeq ($(OS),Darwin)
-	RETRACE_LDFLAGS = $(LDFLAGS) -dylib
+	RETRACE_LDFLAGS = $(LDFLAGS) -dylib -ldl
```

Comment:
Please merge this "if" with the one above. We can easily combine them nicely info one "if".